### DISCUSSION TITLE
commits are not visible on the contribution graph.

### DISCUSSION THREAD

**Discussion by The-CoderBoy** [community/community#67490] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hello
Recently, I created a new repository, and I have done 3 commits on it, but on the contribution graph (green box), only one commit (when the repository was created) shows up. I can see that my repository is updated, but it does not reflect on the contribution graph.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @The-CoderBoy ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. I've gone ahead and moved it for you. Good luck!

**Comment by seyLu** [comment]
Hi @The-CoderBoy,

According to the [Github official docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile):
> [Commit was made less than 24 hours ago](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#commit-was-made-less-than-24-hours-ago)
After making a commit that meets the requirements to count as a contribution, you may need to wait for up to 24 hours to see the contribution appear on your contributions graph.

**Reply by The-CoderBoy** [reply]
Okay, I can see the commits are visible on the contribution graph after 24 hours. Today I also made some commits. Let me check the contribution graph after 24 hours to see if nothing is wrong with my repo settings.

**Reply by The-CoderBoy** [reply]
its been more than 24 hours and still commits are not visible on contribution graph

**Comment by The-CoderBoy** [comment, marked as answer]
I have found the issue. My email address was set to private in the Emails setting. I have set it as public, and then the issue is resolved.
[Github docs](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile#your-local-git-commit-email-isnt-connected-to-your-account)

**Reply by queenofcorgis** [reply]
Nice work - I reopened this Discussion so anyone else who encounters this can easily find your experience to help 👍🏼 Thanks for coming back to let us know the issue and marking as answer!
